app/views/reimbursement/reports/_actions.html.erb

```diff
@@ -196,7 +196,7 @@
           <% admin_tool("overflow-visible") do %>
             <%= link_to reimbursement_report_admin_approve_path(report_id: report.id), class: "btn bg-info", method: :post do %>
               <%= inline_icon "thumbsup" %>
-              Approve <%= report.user.payout_method.title_kind %> Reimbursement
+              Approve <%= report.user.payout_method.title_kind.downcase %> reimbursement
```

Comment:
Can you attach screenshots of this? Want to sanity check that "ACH" is still capitalised